test/functional/api/cas/core.py

```diff
@@ -45,15 +46,34 @@ def __get_core_info(self):
                         "status": split_line[3],
                         "exp_obj": split_line[5]}
 
-    def get_core_statistics(self,
-                            io_class_id: int = None,
+    def get_part_statistics(self,
+                            io_class_id: int,
```

Comment:
Since you have separate method for retrieving ioclass stats, this method shouldn't have `io_class_id` param.